src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,40 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    public static final int BOUND = 20;
+
+    public Figure getRandomFigure() {
+        Figure fig = null;
+        String figure = Figures.values()[new Random().nextInt(Figures.values().length)].toString();
+        switch (figure) {
+            case "ISOSCELESTRAPEZOID" : fig =
+                                      new IsoscelesTrapezoid(new ColorSupplier().getRandomColor(),
+                                      new Random().nextInt(BOUND),
+                                      new Random().nextInt(BOUND),
+                                      new Random().nextInt(BOUND));
```

Comment:
let's create Random outside the switch/case and use this object to create random numbers